pkg/rbd/rbd_util.go

```diff
@@ -26,6 +26,7 @@ import (
 	"time"
 
 	"github.com/golang/glog"
+	"github.com/pkg/errors"
```

Comment:
keep `fmt.Errorf`

Reply:
That doesn't work, because then the root cause of the error is lost and it becomes impossible to distinguish between real, fatal errors and "file not found".